packages/featured-content/stories/FeaturedContentWithDataFetching.stories.tsx

```diff
@@ -201,53 +200,52 @@ export const WithCatalogQuery: FeaturedContentStory = {
   }
 };
 
-const QueryContentsQueryTemplate = () => {
-  const { i18n } = useTranslation();
-  const [addResourceToQueue] = useAddResourceToQueueMutation();
-  const handleAddedToQueue = (item: FeaturedContentContentItem): Promise<void> => {
-    const { slug, kind } = item as FeaturedContentHydratedContentItem;
-    return slug
-      ? addResourceToQueue({
-          variables: {
-            resourceType: kind,
-            resourceId: slug
-          }
-        }).then()
-      : Promise.resolve();
-  };
-
-  const { data, loading, error } = useContentsQuery({
-    variables: { ...mockQueryContentsQueryVariables }
-  });
-  let content;
-  if (loading) {
-    content = <p>Loading content</p>;
-  }
-  if (error) {
-    content = <p>Error loading content</p>;
-  }
-  if (data) {
-    content = data.QueryContents.map((item, index) => {
-      const hydratedItem = hydrateContent(i18n, item);
-      return <ContentTileStandardLayout.Item key={`item-${index}`} {...hydratedItem} />;
-    });
-  }
-  return (
-    <FeaturedContent>
-      <ContentTileStandardLayout
-        headerOptions={headerOptions}
-        desktopColumnCount={3}
-        onAddedToQueue={handleAddedToQueue}
-        onClick={handleClick}
-      >
-        {content}
-      </ContentTileStandardLayout>
-    </FeaturedContent>
-  );
-};
-
 export const WithQueryContentsQuery: FeaturedContentStory = {
-  render: () => <QueryContentsQueryTemplate />,
+  render: () =>
```

Comment:
Wrap the render `() => JSX.Element` with `() => React.createElement(() => JSX.Element)` for using react hooks in the function, this will pass the linting rule of `react-hooks/rules-of-hooks`